src/hooks/useWasVisible.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react'
+import type { MutableRefObject } from 'react'
+
+const useOnceVisible = (element: MutableRefObject<HTMLElement | null>): boolean => {
```

Comment:
Please name the file the same as the hook.

Reply:
renamed